packages/e2e/test/wallet/SingleAddressWallet/byron.test.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import { Cardano } from '@cardano-sdk/core';
+import { SingleAddressWallet } from '@cardano-sdk/wallet';
+import { filter, firstValueFrom, map } from 'rxjs';
+import { getEnv, walletVariables } from '../../environment';
+import { getLogger, getWallet } from '../../../src';
+import { walletReady } from '../../util';
+
+const env = getEnv(walletVariables);
+const logger = getLogger(env.LOGGER_MIN_SEVERITY);
+
+describe('SingleAddressWallet/byron', () => {
+  let wallet: SingleAddressWallet;
+
+  beforeAll(async () => {
+    wallet = (await getWallet({ env, idx: 0, logger, name: 'Wallet', polling: { interval: 50 } })).wallet;
+  });
+
+  afterAll(() => {
+    wallet.shutdown();
+  });
+
+  it('can transfer tADA to a byron address', async () => {
+    await walletReady(wallet);
+
+    const txProps = {
+      outputs: new Set([
+        {
+          address: Cardano.Address('5oP9ib6ym3Xc2XrPGC6S7AaJeHYBCmLjt98bnjKR58xXDhSDgLHr8tht3apMDXf2Mg'),
+          value: {
+            coins: 3_000_000n
+          }
+        }
+      ])
+    };
+
+    const unsignedTx = await wallet.initializeTx(txProps);
```

Comment:
I would propose using the txBuilder because new users searching for examples on how to build transactions will use this as a starting point.